tests/Doctrine/Tests/ORM/Tools/EntityGeneratorTest.php

```diff
@@ -82,6 +82,8 @@ public function generateBookEntityFixture(array $embeddedClasses = array())
         ));
         $metadata->addLifecycleCallback('loading', 'postLoad');
         $metadata->addLifecycleCallback('willBeRemoved', 'preRemove');
+        $metadata->addLifecycleCallback('updateDate', 'preUpdate');
+        $metadata->addLifecycleCallback('updateDate', 'prePersist');
```

Comment:
To be moved to a separate test method that verifies method de-duplication explicitly